github-for-jira.sd.yml

```diff
@@ -287,6 +287,19 @@ environmentOverrides:
           min: 1
           max: 5
           metrics: *CpuMemAndQueuesScalingRules
+    resources:
+      - type: globaledge
+        name: proxy
+        attributes:
+          ip_whitelist:
+            - public
+          routes:
+            - match: # Blackhole IP that keeps spamming us
+                prefix: /
+                external_address_header_match:
+                  - 94.156.174.137
+              route:
+                cluster: blackhole # Deny by sending traffic to blackhole
```

Comment:
Make this global then just set the overrides specific to the env :)

This way we don't have to duplicate the blackhole.

Reply:
Micros don't recommend us make this config global. However I've added a variable to reduce configs copy-pasting